ow_lander/scripts/utils.py

```diff
@@ -18,6 +19,23 @@ def is_shou_yaw_goal_in_range(joint_goal):
 
   if (joint_goal[constants.J_SHOU_YAW]<lower) or (joint_goal[constants.J_SHOU_YAW]>upper):
     return False
-    
   else:
     return True
+
+def _normalize_radians(angle):
+  """
+  :param angle: (float)
+  :return: (float) the angle in [-pi, pi)
+  """
+  # Note: tau = 2 * pi
+  return (angle + pi) % tau - pi
+
+def radians_equivalent (angle1, angle2, tolerance) :
+  return _normalize_radians(abs(angle1 - angle2)) <= tolerance
```

Comment:
I realized what I suggested could cause a bug unless we change up the order of operations. See https://github.com/nasa/ow_autonomy/pull/94#discussion_r1014437678
```suggestion
  return abs(_normalize_radians(angle1 - angle2)) <= tolerance
```